ydkgen/printer/cpp/class_get_namespace_printer.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Siklu
+
+from ydkgen.api_model import Package
+
+
+class GetNamespacePrinter(object):
+    def __init__(self, ctx):
+        self.ctx = ctx
+
+    def print_output(self, clazz):
+        namespace = None
```

Comment:
Maybe better to avoid generating implementation that is identical to the base class' implementation.